warehouse/internal/repo/schema.go

```diff
@@ -231,7 +231,7 @@ func (sh *WHSchema) GetForNamespace(ctx context.Context, destID, namespace strin
 	if err != nil {
 		return model.WHSchema{}, err
 	}
-	if !reflect.DeepEqual(originalSchema.Schema, tableLevelSchemas) {
+	if len(originalSchema.Schema) > 0 && !reflect.DeepEqual(originalSchema.Schema, tableLevelSchemas) {
```

Comment:
If schema is empty, should we do an early return?

Reply:
It should be fine, as we are returning in the next line. We can avoid 1 nested block.